python/podio/reading.py

```diff
@@ -50,32 +50,41 @@ def _determine_root_format(filename):
     return RootFileFormat.RNTUPLE
 
 
-def get_reader(filename):
-    """Get an appropriate reader for the passed file.
+def get_reader(filenames):
+    """Get an appropriate reader for the passed files.
```

Comment:
```suggestion
    """Get an appropriate reader for the passed files.

```
To make API documentation formatted nicely with a brief and long description.